tests/fixtures/mailbox_invalid.txt

```diff
@@ -15,7 +15,6 @@ missing-at-sign.net
 Abc.example.com
 A@b@c@example.com
 a"b(c)d,e:f;g<h>i[j\k]l@example.com
-just"not"right@example.com
```

Comment:
Valid address because we allow `Foo foo@bar.com` without `<>`. Equivalent to `just "not" right@example.com` where `just "not"` is the display name.